build.sbt

```diff
@@ -133,3 +133,5 @@ lazy val uiTests = (project in file("ui-tests"))
   ) dependsOn backend
 
 RenameProject.settings
+
+fork in run := true
```

Comment:
Weird that it could cause problems, but anyway if it's not necessary, no need to add it ;)